shlr/sdb/src/base64.c

```diff
@@ -10,7 +10,7 @@
 static const char cb64[]="ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
 static const char cd64[]="|$$$}rstuvwxyz{$$$$$$$>?@ABCDEFGHIJKLMNOPQRSTUVW$$$$$$XYZ[\\]^_`abcdefghijklmnopq";
 
-static void b64_encode(const ut8 in[3], char out[4], int len) {
+static void sdb_b64_encode(const ut8 in[3], char out[4], int len) {
```

Comment:
i dont think you should update sdb in here, isnt sdb in master already? do you need to rebase maybe?

Reply:
yes, needs a rebase.